src/BuiltInTools/dotnet-watch/HotReload/BlazorWebAssemblyDeltaApplier.cs

```diff
@@ -31,109 +39,31 @@ public override async Task WaitForProcessRunningAsync(CancellationToken cancella
-
-                        await browserRefreshServer.SendAndReceiveAsync(
-                            request: _ => default(JsonGetApplyUpdateCapabilitiesRequest),
-                            response: (value, reporter) =>
-                            {
-                                var str = Encoding.UTF8.GetString(value);
-                                if (str.StartsWith('!'))
-                                {
-                                    reporter.Verbose($"Exception while reading WASM runtime capabilities: {str[1..]}");
-                                }
-                                else if (str.Length == 0)
-                                {
-                                    reporter.Verbose($"Unable to read WASM runtime capabilities");
-                                }
-                                else if (capabilityString == null)
-                                {
-                                    capabilityString = str;
-                                }
-                                else if (capabilityString != str)
-                                {
-                                    reporter.Verbose($"Received different capabilities from different browsers:{Environment.NewLine}'{str}'{Environment.NewLine}'{capabilityString}'");
-                                }
-                            },
-                            cancellationToken);
-
-                        if (capabilityString != null)
-                        {
-                            capabilities = capabilityString;
-                        }
-                        else
-                        {
-                            capabilities = GetDefaultCapabilities(targetFrameworkVersion);
-                            Reporter.Verbose($"Falling back to default WASM capabilities: '{capabilities}'");
-                        }
-                    }
-
-                    // Capabilities are expressed a space-separated string.
-                    // e.g. https://github.com/dotnet/runtime/blob/14343bdc281102bf6fffa1ecdd920221d46761bc/src/coreclr/System.Private.CoreLib/src/System/Reflection/Metadata/AssemblyExtensions.cs#L87
-                    return capabilities.Split(' ').ToImmutableArray();
-                }
-                catch (Exception e) when (!cancellationToken.IsCancellationRequested)
-                {
-                    Reporter.Error($"Failed to read capabilities: {e.Message}");
-
-                    // Do not attempt to retrieve capabilities again if it fails once, unless the operation is canceled.
-                    return [];
-                }
-                finally
-                {
-                    ArrayPool<byte>.Shared.Return(buffer);
-                }
+                    9 => s_defaultCapabilities90,
+                    8 => s_defaultCapabilities80,
+                    7 => s_defaultCapabilities70,
+                    6 => s_defaultCapabilities60,
+                    _ => [],
```

Comment:
Is the minimum TFM for BlazorWasm net6?

Reply:
Yes, I think we have condition elsewhere that requires net6+